jrnl/plugins/util.py

```diff
@@ -3,6 +3,18 @@
 # Copyright (C) 2012-2021 jrnl contributors
 # License: https://www.gnu.org/licenses/gpl-3.0.html
 
+from collections import Counter
+
+
+def get_date_counts(journal):
+    """Returns a collections.Counter object containing date counts"""
+    date_counts = Counter()
+    for entry in journal.entries:
+        # entry.date.date() gets date without time
+        date = str(entry.date.date())
```

Comment:
Maybe this should use the date format from the config file?

Reply:
Isn't the point of a datecount plugin to output the dates universally (so it can be piped elsewhere)? I think you discuss that in #743.